### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 32

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#208] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254597-2a5353ac-48c6-4218-9069-10605ea71e30.png)

**Comment by SvegasTime** [comment]
![Immagine WhatsApp 2024-01-17 ore 16 52 54_90a08b71](https://github.com/sapienzastudentsnetwork/mmi2122/assets/156933897/60ca5458-c928-45d9-9e65-91b74e318965)
